apache-rat-plugin/src/main/java/org/apache/rat/mp/AbstractRatMojo.java

```diff
@@ -655,17 +664,19 @@ private List<IgnoreMatcher> mergeDefaultExclusions() throws MojoExecutionExcepti
                 }
             }
         }
-        if (excludesFile != null) {
-            final File f = new File(excludesFile);
-            if (!f.isFile()) {
-                getLog().error("Excludes file not found: " + f.getAbsolutePath());
-            }
-            if (!f.canRead()) {
-                getLog().error("Excludes file not readable: " + f.getAbsolutePath());
+        if (!excludesFileList.isEmpty()) {
+            for (String fileName : excludesFileList) {
+                final File f = new File(fileName);
+                if (!f.isFile()) {
+                    getLog().error("Excludes file not found: " + f.getAbsolutePath());
+                }
+                if (!f.canRead()) {
+                    getLog().error("Excludes file not readable: " + f.getAbsolutePath());
+                }
+                final String charset = excludesFileCharset == null ? "UTF8" : excludesFileCharset;
```

Comment:
Should this be UTF-8 instead?
https://docs.oracle.com/javase/8/docs/api/java/nio/charset/Charset.html

Reply:
UTF8 is the historical name, changed to "UTF-8" instead after reviewing
https://github.com/AdoptOpenJDK/openjdk-jdk11/blob/master/src/java.base/share/classes/sun/nio/cs/UTF_8.java